src/__tests__/PojoMap.test.ts

```diff
@@ -0,0 +1,17 @@
+import { PojoMap } from '../';
+
+// Leibniz equality test to ensure exact type match
+// see https://medium.com/codestar-blog/leibniz-equality-in-typescript-2aeff1303749
+type Leibniz<A, B> = ((a: A) => B) & ((b: B) => A);
+function identity<T>(t: T): T {
+  return t;
+}
+function leibnizTest<T1, T2>(id: Leibniz<T1, T2>): void {
+  id((undefined as unknown) as T2);
+}
```

Comment:
:point_up: these functions are for doing type-equality tests. They're unused now but I'll be using them later to make some type assertions in other tests.